subsystem/cluster_test.go

```diff
@@ -148,6 +148,21 @@ var _ = Describe("Cluster tests", func() {
 		Expect(c.Hosts[0].ID.String()).Should(Equal(h.ID.String()))
 	})
 
+	It("cluster name exceed max length (54 characters)", func() {
+		_, err1 := userBMClient.Installer.DeregisterCluster(ctx, &installer.DeregisterClusterParams{
+			ClusterID: clusterID,
+		})
+		Expect(err1).ShouldNot(HaveOccurred())
+		cluster, err = userBMClient.Installer.RegisterCluster(ctx, &installer.RegisterClusterParams{
+			NewClusterParams: &models.ClusterCreateParams{
+				Name:             swag.String("abcdefghijabcdefghijabcdefghijabcdefghijabcdefghijabcdefghij"),
+				OpenshiftVersion: swag.String("4.5"),
+				PullSecret:       swag.String(pullSecret),
+			},
+		})
+		Expect(err).Should(HaveOccurred())
```

Comment:
Can we check the error to make sure it's not failing on something else?

Reply:
I think that because of another issue it will return 422 and not 400, but i'm not sure.

Reply:
Anyway the UI have the same validation so it will not affect users that are using  cloud

Reply:
@filanov it really returns 422, what is the issue that causes that?

Reply:
since we are getting an invalid error and the UI validates the same I think we can skip this validation.